### DISCUSSION TITLE
Server stun resistance setting does nothing

### DISCUSSION THREAD

**Discussion by Harry-Wang-dotcom** [FakeFishGames/Barotrauma#14957] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I set server stun resistance setting to 100%, can stun everybody the same time as I used to, and get stunned myself the same time as I used to.

### Reproduction steps

1. start multiplier server
2. set server setting to 100% stun resistance
3. set server setting to spawn some bots if you want
4. start game
5. throw stun grenade on self, see if works
6. use stun baton on a bot, see if works
7. shoot shotgun rubber shells on a bot, see if works

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Linux

**Comment by Jasontti** [comment]
Tested a fix for this.
Closing discussion as resolved.
